src/hotspot/share/vitals/vitals.cpp

```diff
@@ -1214,16 +1254,18 @@ void sample_jvm_values(Sample* sample, bool avoid_locking) {
 
 bool initialize() {
 
-  log_info(os)("Initializing vitals...");
+  log_info(vitals)("Vitals v%x", vitals_version);
+  log_info(vitals)("Initializing vitals...");
 
   // Adjust VitalsSampleInterval
   if (VitalsSampleInterval == 0) {
-    log_warning(os)("Invalid VitalsSampleInterval (" UINTX_FORMAT ") specified. Vitals disabled.",
+    log_warning(vitals)("Invalid VitalsSampleInterval (" UINTX_FORMAT ") specified. Vitals disabled.",
                     VitalsSampleInterval);
     return false;
   }
 
   bool success = ColumnList::initialize();
+  success &= Legend::initialize();
 
   // Order matters. First platform columns, then jvm columns.
   success &= platform_columns_initialize();
```

Comment:
I don't think so, since we only execute platform_columns_initialize() if success was true before, and its only true if the new succeeded. (I do this &= trick when I am too lazy to do individual error checks, or to keep the code small).

Reply:
But the return value is wrong, should return success.